docs/source/_templates/navbar.html

```diff
@@ -1,45 +0,0 @@
-<div id="navbar" class="{{ theme_navbar_class }} navbar-default {% if theme_navbar_fixed_top|tobool -%} navbar-fixed-top{%- endif -%}">
-  <div class="container">
-    <div class="navbar-header">
-      <!-- .btn-navbar is used as the toggle for collapsed navbar content -->
-      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".nav-collapse">
-        <span class="icon-bar"></span>
-        <span class="icon-bar"></span>
-        <span class="icon-bar"></span>
-      </button>
-      <a class="navbar-brand" href="{{ pathto('index') }}">
-        {%- block sidebarlogo %}
-          {%- if logo %}<img src="{{ pathto('_static/' + logo, 1) }}">{%- endif %}
```

Comment:
Oh yeah and the logo is added here.... why ? We have a main "jupyter-sphinx-theme", the logo should be in the main theme. 

And also the logo is not in the right location, i t should be in the `div.col-md-3` to have the right width.